admin-notice/class-admin-notice.php

```diff
@@ -5,20 +5,35 @@
 class Admin_Notice {
 	public $message;
 	public $conditions;
+	public $dismiss_identifier;
+
+	const COOKIE_DELIMETER = '|';
 
 	/**
 	 * Create AdminNotice.
 	 *
 	 * @param string $message The text to be displayed.
 	 * @param Admin_Notice_Condition[] $conditions Conditions to pass in order to show the notice.
+	 * @param string $dismiss_identifier if provided the dismiss will become dissmisible
 	 */
-	public function __construct( string $message, array $conditions = [] ) {
+	public function __construct( string $message, array $conditions = [], string $dismiss_identifier = '' ) {
+		$position = strpos( $dismiss_identifier, self::COOKIE_DELIMETER );
+		if ( false !== $position ) {
+			\trigger_error( esc_html( "Admin Notice dissmiss identifier - $dismiss_identifier can't contain delimeter - " . self::COOKIE_DELIMETER ) );
```

Comment:
Not sure what to do here? Should I log error in another way?

Reply:
To log it into Kibana, it needs the second arg passed to bump the default error level.

Reply:
What should be the correct second arg or how do I find out?

Reply:
E_USER_ERROR?

Reply:
It's in our public docs on WPVIP.com.

Reply:
Found it thanks! - https://wpvip.com/documentation/manually-logging-errors-in-new-relic/ => `E_USER_WARNING`